js/everything.js

```diff
@@ -280,6 +347,31 @@ var ZenPlayer = {
             toggleElement(event, ".plyr__video-wrapper", "Player");
         });
     },
+    setupAutoplayToggle: function()
+	{
+		// toggle auto next song playing
+        $("#toggleAutoplay").click(function(event)
+		{
+            var toggleTextElement = $("#" + event.currentTarget.id);
+            if (autoplayState === null)
+			{
+                autoplayState = "true";
```

Comment:
Mostly because session storage wont allow non-strings to be stored, so just wanted to avoid the conversion from bool to string at multiple places.